packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -102,11 +105,16 @@ class AppBar extends StatelessWidget {
   /// The following elevations have defined shadows: 1, 2, 3, 4, 6, 8, 9, 12, 16, 24
   final int elevation;
 
-  /// The color to use for the the app bar's material.
+  /// The color to use for the app bar's material.
   ///
   /// Defaults to [ThemeData.primaryColor].
   final Color backgroundColor;
 
+  /// The brightness of the app bar's material.
```

Comment:
This comment (and the one for backgroundColor) should probably mention that the two parameters should be set in tandem.